apache/uima-uimaj
# Refactoring/331 convert remaining documentation to asciidoc

**What's in the PR**
* Update all documentation to be Asciidoctor
* Consolidate all documentation in a single module

**How to test manually**
* Run a fake release build and check out the generated artifacts (`mvn clean install -Papache-release -Ddisable-rc-auto-staging -DskipTests`)

**Automatic testing**

**Documentation**

**Organizational**
      <sub><sup>Only dependencies under [approved licenses](http://www.apache.org/legal/resolved.html#category-a) are allowed. LICENSE and NOTICE files in the respective modules where dependencies have been added as well as in the project root have been updated.</sup></sub>
